text/0075-fast-sync.md

```diff
@@ -0,0 +1,155 @@
+# Fast Synchronization
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0000](https://github.com/casperlabs/ceps/pull/0000)
+
+Joining nodes avoid executing every block by downloading the database
+instead. This is *fast syncing*.
+
+## Motivation
+
+[motivation]: #motivation
+
+For a long chain, it is faster to download the database than execute
+every block. The chain's database contains account balances. It also
+contains the smart contracts and their state.
+
+Nodes only need recent database states to operate, so it saves disk
+space to fast sync.
+
+Fast syncing avoids the need to be backward compatible with old
+versions. This is because the current executable may not support
+executing old blocks. New versions need to support old executables
+downloading blocks.
+
+After implementation nodes will join the network by fast syncing.
+Operators will only need the current to join the network.
```

Comment:
```suggestion
Operators will only need the current version to join the network.
```

Reply:
Done in https://github.com/casper-network/ceps/pull/75/commits/a43ffc357c3c606c0cf7cfe65f5150c1bb408b46.